playbooks/webapp/vars/main.yaml

```diff
@@ -55,3 +55,21 @@ deploy_flask_app_local_registry_pwd: testing123
 deploy_flask_app_local_registry_port: "{{ deploy_flask_app_listening_port }}"
 deploy_flask_app_rds_master_password: L#5cH2mgy_
 deploy_flask_app_rds_master_username: ansible
+
+# vars to create aurora db cluster
+# Global cluster parameters ================================
+rds_global_cluster_name: "{{ resource_prefix }}-global-cluster"
+
+# Primary cluster parameters ================================
+rds_primary_cluster_name: "{{ resource_prefix }}-primary-cluster"
+rds_primary_cluster_region: us-west-2
+rds_primary_cluster_instance_name: "{{ resource_prefix }}-primary-instance"
+
+# Replica cluster parameters ================================
+rds_replica_cluster_name: "{{ resource_prefix }}-replica-cluster"
+rds_replica_cluster_region: us-east-2
+rds_replica_cluster_instance_name: "{{ resource_prefix }}-replica-instance"
+
+# vars for route53 records
+route53_zone_name: "ansiblecloud.xyz"
```

Comment:
I've updated the README to specify that route53_zone_name doesn't have a default value. I'm uncertain whether I should remove this variable from the vars file or leave it as is.

Reply:
It effectively functions as a default value if it's still in the vars file since that value will get used if the user doesn't provide it themselves, so I would lean towards removing it.